c_src/ex_libsrt/srt_nif.cpp

```diff
@@ -270,7 +270,7 @@ start_client(UnifexEnv* env, char* server_address, int port, char* stream_id) {
       send_srt_client_error(state->env, state->owner, 1, reason.c_str());
     });
 
-    state->client->Run(server_address, port, stream_id);
+    state->client->Run(server_address, port, stream_id, std::string(password));
```

Comment:
does `std::string` create a copy of a password? We store it in the client's state to use it further and I believe that `char* password`  argument is destroyed when this function returns.